### PR TITLE
Rearranged the order of execution of test suites

### PR DESCRIPTION
**Pull request by PavamanSubramaniyam**
Created the new different driver cfg files covering their respective driver module parameter tests

### CODE REVIEW

**Comment by PavamanSubramaniyam** [comment]
Netperf tests with --execution-order tests-per-variant option shows execution of 45 tests

ltcever60-lp8:~/avocado-fvt-wrapper/tests/avocado-misc-tests/io/net # cat netperf_test.py.data/netperf_test.yaml
interface: "eth0"
peer_ip: "17.1.1.28"
peer_public_ip: "9.3.101.28"
host_ip: "17.1.1.29"
netmask: "255.255.255.0"
peer_user: "root"
peer_password: "llm@fvt"
TIMEOUT: "720"
NETSERVER_RUN: 1
EXPECTED_THROUGHPUT: 90
duration: 120
minimum_iterations: 1
maximum_iterations: 5
netperf_download: "https://github.com/HewlettPackard/netperf/archive/netperf-2.7.0.zip"
option: !mux
    generic:
        option: ''
    tcp_stream:
        option: 'TCP_STREAM -- -m 63000'
    udp_stream:
        option: 'UDP_STREAM -- -m 63000'
    tcp_rr:
        option: 'TCP_RR'
    udp_rr:
        option: 'UDP_RR'
mtu: !mux
    1500:
        mtu: "1500"
    2000:
        mtu: "2000"
    3000:
        mtu: "3000"
    4000:
        mtu: "4000"
    5000:
        mtu: "5000"
    6000:
        mtu: "6000"
    7000:
        mtu: "7000"
    8000:
        mtu: "8000"
    9000:
        mtu: "9000"
ltcever60-lp8:~/avocado-fvt-wrapper/tests/avocado-misc-tests/io/net # avocado run --max-parallel-tasks=1 netperf_test.py -m netperf_test.py.data/netperf_test.yaml --execution-order tests-per-variant
JOB ID     : c1e50af3a7d041b3ef7c04a7a2c0a40ead190569
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2023-03-21T00.57-c1e50af/job.log
Task exception was never retrieved
future: <Task finished coro=<StatusServer.serve_forever() done, defined at /usr/lib/python3.6/site-packages/avocado_framework-100.1-py3.6.egg/avocado/core/status/server.py:40> exception=AttributeError("'Server' object has no attribute 'serve_forever'",)>
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/avocado_framework-100.1-py3.6.egg/avocado/core/status/server.py", line 44, in serve_forever
    await self._server_task.serve_forever()
AttributeError: 'Server' object has no attribute 'serve_forever'
 (01/45) netperf_test.py:Netperf.test;run-mtu-1500-option-generic-8b0b: STARTED
 (01/45) netperf_test.py:Netperf.test;run-mtu-1500-option-generic-8b0b: PASS (377.33 s)
 (02/45) netperf_test.py:Netperf.test;run-mtu-2000-option-generic-a3f5: STARTED
 (02/45) netperf_test.py:Netperf.test;run-mtu-2000-option-generic-a3f5: PASS (377.63 s)
 (03/45) netperf_test.py:Netperf.test;run-mtu-3000-option-generic-e9b5: STARTED
 (03/45) netperf_test.py:Netperf.test;run-mtu-3000-option-generic-e9b5: PASS (378.77 s)
 (04/45) netperf_test.py:Netperf.test;run-mtu-4000-option-generic-16fb: STARTED
 (04/45) netperf_test.py:Netperf.test;run-mtu-4000-option-generic-16fb: PASS (377.58 s)
 (05/45) netperf_test.py:Netperf.test;run-mtu-5000-option-generic-a5f7: STARTED
 (05/45) netperf_test.py:Netperf.test;run-mtu-5000-option-generic-a5f7: PASS (378.82 s)
 (06/45) netperf_test.py:Netperf.test;run-mtu-6000-option-generic-5587: STARTED
 (06/45) netperf_test.py:Netperf.test;run-mtu-6000-option-generic-5587: PASS (377.65 s)
 (07/45) netperf_test.py:Netperf.test;run-mtu-7000-option-generic-5c5d: STARTED
 (07/45) netperf_test.py:Netperf.test;run-mtu-7000-option-generic-5c5d: PASS (378.88 s)
 (08/45) netperf_test.py:Netperf.test;run-mtu-8000-option-generic-2539: STARTED
 (08/45) netperf_test.py:Netperf.test;run-mtu-8000-option-generic-2539: PASS (377.43 s)
 (09/45) netperf_test.py:Netperf.test;run-mtu-9000-option-generic-559d: STARTED
 (09/45) netperf_test.py:Netperf.test;run-mtu-9000-option-generic-559d: PASS (378.70 s)
 (10/45) netperf_test.py:Netperf.test;run-mtu-1500-option-tcp_stream-70b2: STARTED
 (10/45) netperf_test.py:Netperf.test;run-mtu-1500-option-tcp_stream-70b2: PASS (25.16 s)
 (11/45) netperf_test.py:Netperf.test;run-mtu-2000-option-tcp_stream-fd87: STARTED
 (11/45) netperf_test.py:Netperf.test;run-mtu-2000-option-tcp_stream-fd87: PASS (27.34 s)
 (12/45) netperf_test.py:Netperf.test;run-mtu-3000-option-tcp_stream-9137: STARTED
 (12/45) netperf_test.py:Netperf.test;run-mtu-3000-option-tcp_stream-9137: PASS (27.70 s)
 (13/45) netperf_test.py:Netperf.test;run-mtu-4000-option-tcp_stream-188e: STARTED
 (13/45) netperf_test.py:Netperf.test;run-mtu-4000-option-tcp_stream-188e: PASS (27.60 s)
 (14/45) netperf_test.py:Netperf.test;run-mtu-5000-option-tcp_stream-24d4: STARTED
 (14/45) netperf_test.py:Netperf.test;run-mtu-5000-option-tcp_stream-24d4: PASS (27.46 s)
 (15/45) netperf_test.py:Netperf.test;run-mtu-6000-option-tcp_stream-eb06: STARTED
 (15/45) netperf_test.py:Netperf.test;run-mtu-6000-option-tcp_stream-eb06: PASS (27.51 s)
 (16/45) netperf_test.py:Netperf.test;run-mtu-7000-option-tcp_stream-3ed8: STARTED
 (16/45) netperf_test.py:Netperf.test;run-mtu-7000-option-tcp_stream-3ed8: PASS (27.39 s)
 (17/45) netperf_test.py:Netperf.test;run-mtu-8000-option-tcp_stream-c307: STARTED
 (17/45) netperf_test.py:Netperf.test;run-mtu-8000-option-tcp_stream-c307: PASS (27.33 s)
 (18/45) netperf_test.py:Netperf.test;run-mtu-9000-option-tcp_stream-9dbc: STARTED
 (18/45) netperf_test.py:Netperf.test;run-mtu-9000-option-tcp_stream-9dbc: PASS (28.71 s)
 (19/45) netperf_test.py:Netperf.test;run-mtu-1500-option-udp_stream-54b0: STARTED
 (19/45) netperf_test.py:Netperf.test;run-mtu-1500-option-udp_stream-54b0: PASS (25.23 s)
 (20/45) netperf_test.py:Netperf.test;run-mtu-2000-option-udp_stream-32b1: STARTED
 (20/45) netperf_test.py:Netperf.test;run-mtu-2000-option-udp_stream-32b1: PASS (27.71 s)
 (21/45) netperf_test.py:Netperf.test;run-mtu-3000-option-udp_stream-ebb3: STARTED
 (21/45) netperf_test.py:Netperf.test;run-mtu-3000-option-udp_stream-ebb3: PASS (27.56 s)
 (22/45) netperf_test.py:Netperf.test;run-mtu-4000-option-udp_stream-93fb: STARTED
 (22/45) netperf_test.py:Netperf.test;run-mtu-4000-option-udp_stream-93fb: PASS (27.44 s)
 (23/45) netperf_test.py:Netperf.test;run-mtu-5000-option-udp_stream-a0f7: STARTED
 (23/45) netperf_test.py:Netperf.test;run-mtu-5000-option-udp_stream-a0f7: PASS (27.31 s)
 (24/45) netperf_test.py:Netperf.test;run-mtu-6000-option-udp_stream-597e: STARTED
 (24/45) netperf_test.py:Netperf.test;run-mtu-6000-option-udp_stream-597e: PASS (27.43 s)
 (25/45) netperf_test.py:Netperf.test;run-mtu-7000-option-udp_stream-f7e8: STARTED
 (25/45) netperf_test.py:Netperf.test;run-mtu-7000-option-udp_stream-f7e8: PASS (27.44 s)
 (26/45) netperf_test.py:Netperf.test;run-mtu-8000-option-udp_stream-b900: STARTED
 (26/45) netperf_test.py:Netperf.test;run-mtu-8000-option-udp_stream-b900: PASS (27.40 s)
 (27/45) netperf_test.py:Netperf.test;run-mtu-9000-option-udp_stream-f396: STARTED
 (27/45) netperf_test.py:Netperf.test;run-mtu-9000-option-udp_stream-f396: PASS (27.48 s)
 (28/45) netperf_test.py:Netperf.test;run-mtu-1500-option-tcp_rr-b0a6: STARTED
 (28/45) netperf_test.py:Netperf.test;run-mtu-1500-option-tcp_rr-b0a6: PASS (495.15 s)
 (29/45) netperf_test.py:Netperf.test;run-mtu-2000-option-tcp_rr-e91d: STARTED
 (29/45) netperf_test.py:Netperf.test;run-mtu-2000-option-tcp_rr-e91d: WARN: Test passed but there were warnings during execution. Check the log for details. (618.66 s)
 (30/45) netperf_test.py:Netperf.test;run-mtu-3000-option-tcp_rr-fce2: STARTED
 (30/45) netperf_test.py:Netperf.test;run-mtu-3000-option-tcp_rr-fce2: WARN: Test passed but there were warnings during execution. Check the log for details. (618.51 s)
 (31/45) netperf_test.py:Netperf.test;run-mtu-4000-option-tcp_rr-0a6b: STARTED
 (31/45) netperf_test.py:Netperf.test;run-mtu-4000-option-tcp_rr-0a6b: PASS (378.59 s)
 (32/45) netperf_test.py:Netperf.test;run-mtu-5000-option-tcp_rr-6f9f: STARTED
 (32/45) netperf_test.py:Netperf.test;run-mtu-5000-option-tcp_rr-6f9f: WARN: Test passed but there were warnings during execution. Check the log for details. (617.26 s)
 (33/45) netperf_test.py:Netperf.test;run-mtu-6000-option-tcp_rr-d400: STARTED
 (33/45) netperf_test.py:Netperf.test;run-mtu-6000-option-tcp_rr-d400: PASS (618.59 s)
 (34/45) netperf_test.py:Netperf.test;run-mtu-7000-option-tcp_rr-358e: STARTED
 (34/45) netperf_test.py:Netperf.test;run-mtu-7000-option-tcp_rr-358e: WARN: Test passed but there were warnings during execution. Check the log for details. (618.52 s)
 (35/45) netperf_test.py:Netperf.test;run-mtu-8000-option-tcp_rr-2682: STARTED
 (35/45) netperf_test.py:Netperf.test;run-mtu-8000-option-tcp_rr-2682: PASS (498.88 s)
 (36/45) netperf_test.py:Netperf.test;run-mtu-9000-option-tcp_rr-30b8: STARTED
 (36/45) netperf_test.py:Netperf.test;run-mtu-9000-option-tcp_rr-30b8: PASS (377.37 s)
 (37/45) netperf_test.py:Netperf.test;run-mtu-1500-option-udp_rr-9f64: STARTED
 (37/45) netperf_test.py:Netperf.test;run-mtu-1500-option-udp_rr-9f64: WARN: Test passed but there were warnings during execution. Check the log for details. (616.35 s)
 (38/45) netperf_test.py:Netperf.test;run-mtu-2000-option-udp_rr-547e: STARTED
 (38/45) netperf_test.py:Netperf.test;run-mtu-2000-option-udp_rr-547e: WARN: Test passed but there were warnings during execution. Check the log for details. (618.66 s)
 (39/45) netperf_test.py:Netperf.test;run-mtu-3000-option-udp_rr-880b: STARTED
 (39/45) netperf_test.py:Netperf.test;run-mtu-3000-option-udp_rr-880b: WARN: Test passed but there were warnings during execution. Check the log for details. (618.79 s)
 (40/45) netperf_test.py:Netperf.test;run-mtu-4000-option-udp_rr-58ff: STARTED
 (40/45) netperf_test.py:Netperf.test;run-mtu-4000-option-udp_rr-58ff: WARN: Test passed but there were warnings during execution. Check the log for details. (618.92 s)
 (41/45) netperf_test.py:Netperf.test;run-mtu-5000-option-udp_rr-3d91: STARTED
 (41/45) netperf_test.py:Netperf.test;run-mtu-5000-option-udp_rr-3d91: WARN: Test passed but there were warnings during execution. Check the log for details. (618.61 s)
 (42/45) netperf_test.py:Netperf.test;run-mtu-6000-option-udp_rr-7010: STARTED
 (42/45) netperf_test.py:Netperf.test;run-mtu-6000-option-udp_rr-7010: WARN: Test passed but there were warnings during execution. Check the log for details. (618.60 s)
 (43/45) netperf_test.py:Netperf.test;run-mtu-7000-option-udp_rr-e904: STARTED
 (43/45) netperf_test.py:Netperf.test;run-mtu-7000-option-udp_rr-e904: WARN: Test passed but there were warnings during execution. Check the log for details. (618.61 s)
 (44/45) netperf_test.py:Netperf.test;run-mtu-8000-option-udp_rr-1b10: STARTED
 (44/45) netperf_test.py:Netperf.test;run-mtu-8000-option-udp_rr-1b10: WARN: Test passed but there were warnings during execution. Check the log for details. (618.59 s)
 (45/45) netperf_test.py:Netperf.test;run-mtu-9000-option-udp_rr-d233: STARTED
 (45/45) netperf_test.py:Netperf.test;run-mtu-9000-option-udp_rr-d233: PASS (378.59 s)
RESULTS    : PASS 33 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 12 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2023-03-21T00.57-c1e50af/results.html
JOB TIME   : 14106.71 s

**Comment by PavamanSubramaniyam** [comment]
Netperf tests without the --execution-order tests-per-variant option also, I see the same 45 tests getting executed

ltcever60-lp8:~/avocado-fvt-wrapper/tests/avocado-misc-tests/io/net # cat netperf_test.py.data/netperf_test.yaml
interface: "eth0"
peer_ip: "17.1.1.28"
peer_public_ip: "9.3.101.28"
host_ip: "17.1.1.29"
netmask: "255.255.255.0"
peer_user: "root"
peer_password: "llm@fvt"
TIMEOUT: "720"
NETSERVER_RUN: 1
EXPECTED_THROUGHPUT: 90
duration: 120
minimum_iterations: 1
maximum_iterations: 5
netperf_download: "https://github.com/HewlettPackard/netperf/archive/netperf-2.7.0.zip"
option: !mux
    generic:
        option: ''
    tcp_stream:
        option: 'TCP_STREAM -- -m 63000'
    udp_stream:
        option: 'UDP_STREAM -- -m 63000'
    tcp_rr:
        option: 'TCP_RR'
    udp_rr:
        option: 'UDP_RR'
mtu: !mux
    1500:
        mtu: "1500"
    2000:
        mtu: "2000"
    3000:
        mtu: "3000"
    4000:
        mtu: "4000"
    5000:
        mtu: "5000"
    6000:
        mtu: "6000"
    7000:
        mtu: "7000"
    8000:
        mtu: "8000"
    9000:
        mtu: "9000"
ltcever60-lp8:~/avocado-fvt-wrapper/tests/avocado-misc-tests/io/net # avocado run --max-parallel-tasks=1 netperf_test.py -m netperf_test.py.data/netperf_test.yaml
JOB ID     : 286e9e58b8252eba502516c7822785b3d7e2f4c0
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2023-03-26T02.03-286e9e5/job.log
Task exception was never retrieved
future: <Task finished coro=<StatusServer.serve_forever() done, defined at /usr/lib/python3.6/site-packages/avocado_framework-100.1-py3.6.egg/avocado/core/status/server.py:40> exception=AttributeError("'Server' object has no attribute 'serve_forever'",)>
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/avocado_framework-100.1-py3.6.egg/avocado/core/status/server.py", line 44, in serve_forever
    await self._server_task.serve_forever()
AttributeError: 'Server' object has no attribute 'serve_forever'
 (01/45) netperf_test.py:Netperf.test;run-mtu-1500-option-generic-8b0b: STARTED
 (01/45) netperf_test.py:Netperf.test;run-mtu-1500-option-generic-8b0b: PASS (373.76 s)
 (02/45) netperf_test.py:Netperf.test;run-mtu-2000-option-generic-a3f5: STARTED
 (02/45) netperf_test.py:Netperf.test;run-mtu-2000-option-generic-a3f5: PASS (376.01 s)
 (03/45) netperf_test.py:Netperf.test;run-mtu-3000-option-generic-e9b5: STARTED
 (03/45) netperf_test.py:Netperf.test;run-mtu-3000-option-generic-e9b5: PASS (375.93 s)
 (04/45) netperf_test.py:Netperf.test;run-mtu-4000-option-generic-16fb: STARTED
 (04/45) netperf_test.py:Netperf.test;run-mtu-4000-option-generic-16fb: PASS (376.45 s)
 (05/45) netperf_test.py:Netperf.test;run-mtu-5000-option-generic-a5f7: STARTED
 (05/45) netperf_test.py:Netperf.test;run-mtu-5000-option-generic-a5f7: PASS (375.99 s)
 (06/45) netperf_test.py:Netperf.test;run-mtu-6000-option-generic-5587: STARTED
 (06/45) netperf_test.py:Netperf.test;run-mtu-6000-option-generic-5587: PASS (375.93 s)
 (07/45) netperf_test.py:Netperf.test;run-mtu-7000-option-generic-5c5d: STARTED
 (07/45) netperf_test.py:Netperf.test;run-mtu-7000-option-generic-5c5d: PASS (376.38 s)
 (08/45) netperf_test.py:Netperf.test;run-mtu-8000-option-generic-2539: STARTED
 (08/45) netperf_test.py:Netperf.test;run-mtu-8000-option-generic-2539: PASS (376.20 s)
 (09/45) netperf_test.py:Netperf.test;run-mtu-9000-option-generic-559d: STARTED
 (09/45) netperf_test.py:Netperf.test;run-mtu-9000-option-generic-559d: PASS (376.03 s)
 (10/45) netperf_test.py:Netperf.test;run-mtu-1500-option-tcp_stream-70b2: STARTED
 (10/45) netperf_test.py:Netperf.test;run-mtu-1500-option-tcp_stream-70b2: PASS (23.56 s)
 (11/45) netperf_test.py:Netperf.test;run-mtu-2000-option-tcp_stream-fd87: STARTED
 (11/45) netperf_test.py:Netperf.test;run-mtu-2000-option-tcp_stream-fd87: PASS (25.72 s)
 (12/45) netperf_test.py:Netperf.test;run-mtu-3000-option-tcp_stream-9137: STARTED
 (12/45) netperf_test.py:Netperf.test;run-mtu-3000-option-tcp_stream-9137: PASS (25.71 s)
 (13/45) netperf_test.py:Netperf.test;run-mtu-4000-option-tcp_stream-188e: STARTED
 (13/45) netperf_test.py:Netperf.test;run-mtu-4000-option-tcp_stream-188e: PASS (25.83 s)
 (14/45) netperf_test.py:Netperf.test;run-mtu-5000-option-tcp_stream-24d4: STARTED
 (14/45) netperf_test.py:Netperf.test;run-mtu-5000-option-tcp_stream-24d4: PASS (25.71 s)
 (15/45) netperf_test.py:Netperf.test;run-mtu-6000-option-tcp_stream-eb06: STARTED
 (15/45) netperf_test.py:Netperf.test;run-mtu-6000-option-tcp_stream-eb06: PASS (26.44 s)
 (16/45) netperf_test.py:Netperf.test;run-mtu-7000-option-tcp_stream-3ed8: STARTED
 (16/45) netperf_test.py:Netperf.test;run-mtu-7000-option-tcp_stream-3ed8: PASS (25.75 s)
 (17/45) netperf_test.py:Netperf.test;run-mtu-8000-option-tcp_stream-c307: STARTED
 (17/45) netperf_test.py:Netperf.test;run-mtu-8000-option-tcp_stream-c307: PASS (25.73 s)
 (18/45) netperf_test.py:Netperf.test;run-mtu-9000-option-tcp_stream-9dbc: STARTED
 (18/45) netperf_test.py:Netperf.test;run-mtu-9000-option-tcp_stream-9dbc: PASS (25.75 s)
 (19/45) netperf_test.py:Netperf.test;run-mtu-1500-option-udp_stream-54b0: STARTED
 (19/45) netperf_test.py:Netperf.test;run-mtu-1500-option-udp_stream-54b0: PASS (23.62 s)
 (20/45) netperf_test.py:Netperf.test;run-mtu-2000-option-udp_stream-32b1: STARTED
 (20/45) netperf_test.py:Netperf.test;run-mtu-2000-option-udp_stream-32b1: PASS (26.07 s)
 (21/45) netperf_test.py:Netperf.test;run-mtu-3000-option-udp_stream-ebb3: STARTED
 (21/45) netperf_test.py:Netperf.test;run-mtu-3000-option-udp_stream-ebb3: PASS (25.71 s)
 (22/45) netperf_test.py:Netperf.test;run-mtu-4000-option-udp_stream-93fb: STARTED
 (22/45) netperf_test.py:Netperf.test;run-mtu-4000-option-udp_stream-93fb: PASS (25.77 s)
 (23/45) netperf_test.py:Netperf.test;run-mtu-5000-option-udp_stream-a0f7: STARTED
 (23/45) netperf_test.py:Netperf.test;run-mtu-5000-option-udp_stream-a0f7: PASS (25.75 s)
 (24/45) netperf_test.py:Netperf.test;run-mtu-6000-option-udp_stream-597e: STARTED
 (24/45) netperf_test.py:Netperf.test;run-mtu-6000-option-udp_stream-597e: PASS (25.76 s)
 (25/45) netperf_test.py:Netperf.test;run-mtu-7000-option-udp_stream-f7e8: STARTED
 (25/45) netperf_test.py:Netperf.test;run-mtu-7000-option-udp_stream-f7e8: PASS (25.73 s)
 (26/45) netperf_test.py:Netperf.test;run-mtu-8000-option-udp_stream-b900: STARTED
 (26/45) netperf_test.py:Netperf.test;run-mtu-8000-option-udp_stream-b900: PASS (25.92 s)
 (27/45) netperf_test.py:Netperf.test;run-mtu-9000-option-udp_stream-f396: STARTED
 (27/45) netperf_test.py:Netperf.test;run-mtu-9000-option-udp_stream-f396: PASS (26.01 s)
 (28/45) netperf_test.py:Netperf.test;run-mtu-1500-option-tcp_rr-b0a6: STARTED
 (28/45) netperf_test.py:Netperf.test;run-mtu-1500-option-tcp_rr-b0a6: WARN: Test passed but there were warnings during execution. Check the log for details. (613.58 s)
 (29/45) netperf_test.py:Netperf.test;run-mtu-2000-option-tcp_rr-e91d: STARTED
 (29/45) netperf_test.py:Netperf.test;run-mtu-2000-option-tcp_rr-e91d: WARN: Test passed but there were warnings during execution. Check the log for details. (616.23 s)
 (30/45) netperf_test.py:Netperf.test;run-mtu-3000-option-tcp_rr-fce2: STARTED
 (30/45) netperf_test.py:Netperf.test;run-mtu-3000-option-tcp_rr-fce2: PASS (495.67 s)
 (31/45) netperf_test.py:Netperf.test;run-mtu-4000-option-tcp_rr-0a6b: STARTED
 (31/45) netperf_test.py:Netperf.test;run-mtu-4000-option-tcp_rr-0a6b: WARN: Test passed but there were warnings during execution. Check the log for details. (615.85 s)
 (32/45) netperf_test.py:Netperf.test;run-mtu-5000-option-tcp_rr-6f9f: STARTED
 (32/45) netperf_test.py:Netperf.test;run-mtu-5000-option-tcp_rr-6f9f: WARN: Test passed but there were warnings during execution. Check the log for details. (615.91 s)
 (33/45) netperf_test.py:Netperf.test;run-mtu-6000-option-tcp_rr-d400: STARTED
 (33/45) netperf_test.py:Netperf.test;run-mtu-6000-option-tcp_rr-d400: WARN: Test passed but there were warnings during execution. Check the log for details. (615.68 s)
 (34/45) netperf_test.py:Netperf.test;run-mtu-7000-option-tcp_rr-358e: STARTED
 (34/45) netperf_test.py:Netperf.test;run-mtu-7000-option-tcp_rr-358e: WARN: Test passed but there were warnings during execution. Check the log for details. (616.27 s)
 (35/45) netperf_test.py:Netperf.test;run-mtu-8000-option-tcp_rr-2682: STARTED
 (35/45) netperf_test.py:Netperf.test;run-mtu-8000-option-tcp_rr-2682: WARN: Test passed but there were warnings during execution. Check the log for details. (615.69 s)
 (36/45) netperf_test.py:Netperf.test;run-mtu-9000-option-tcp_rr-30b8: STARTED
 (36/45) netperf_test.py:Netperf.test;run-mtu-9000-option-tcp_rr-30b8: PASS (615.94 s)
 (37/45) netperf_test.py:Netperf.test;run-mtu-1500-option-udp_rr-9f64: STARTED
 (37/45) netperf_test.py:Netperf.test;run-mtu-1500-option-udp_rr-9f64: WARN: Test passed but there were warnings during execution. Check the log for details. (613.54 s)
 (38/45) netperf_test.py:Netperf.test;run-mtu-2000-option-udp_rr-547e: STARTED
 (38/45) netperf_test.py:Netperf.test;run-mtu-2000-option-udp_rr-547e: WARN: Test passed but there were warnings during execution. Check the log for details. (615.70 s)
 (39/45) netperf_test.py:Netperf.test;run-mtu-3000-option-udp_rr-880b: STARTED
 (39/45) netperf_test.py:Netperf.test;run-mtu-3000-option-udp_rr-880b: PASS (496.08 s)
 (40/45) netperf_test.py:Netperf.test;run-mtu-4000-option-udp_rr-58ff: STARTED
 (40/45) netperf_test.py:Netperf.test;run-mtu-4000-option-udp_rr-58ff: PASS (615.71 s)
 (41/45) netperf_test.py:Netperf.test;run-mtu-5000-option-udp_rr-3d91: STARTED
 (41/45) netperf_test.py:Netperf.test;run-mtu-5000-option-udp_rr-3d91: PASS (376.00 s)
 (42/45) netperf_test.py:Netperf.test;run-mtu-6000-option-udp_rr-7010: STARTED
 (42/45) netperf_test.py:Netperf.test;run-mtu-6000-option-udp_rr-7010: WARN: Test passed but there were warnings during execution. Check the log for details. (615.74 s)
 (43/45) netperf_test.py:Netperf.test;run-mtu-7000-option-udp_rr-e904: STARTED
 (43/45) netperf_test.py:Netperf.test;run-mtu-7000-option-udp_rr-e904: PASS (495.75 s)
 (44/45) netperf_test.py:Netperf.test;run-mtu-8000-option-udp_rr-1b10: STARTED
 (44/45) netperf_test.py:Netperf.test;run-mtu-8000-option-udp_rr-1b10: PASS (376.02 s)
 (45/45) netperf_test.py:Netperf.test;run-mtu-9000-option-udp_rr-d233: STARTED
 (45/45) netperf_test.py:Netperf.test;run-mtu-9000-option-udp_rr-d233: WARN: Test passed but there were warnings during execution. Check the log for details. (616.11 s)
RESULTS    : PASS 34 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 11 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2023-03-26T02.03-286e9e5/results.html
JOB TIME   : 14130.79 s

**Comment by PavamanSubramaniyam** [comment]
As can be seen with and without the --execution order option all the MTU sizes and mux combination of tests are getting executed as expected. So, I don't find any difference in the results which I have pasted here.

**Comment by abdhaleegit** [comment]
LGTM

File: config/tests/host/io_network.cfg [hunk 1]
```diff
@@ -1,22 +1,23 @@
 avocado-misc-tests/io/common/bootlist_test.py avocado-misc-tests/io/common/bootlist_test.py.data/bootlist_test_network.yaml
-avocado-misc-tests/io/net/ethtool_test.py avocado-misc-tests/io/net/ethtool_test.py.data/ethtool_test.yaml
+avocado-misc-tests/io/common/distro_tools.py avocado-misc-tests/io/common/distro_tools.py.data/distro_tools_common.yaml
+avocado-misc-tests/io/common/distro_tools.py avocado-misc-tests/io/common/distro_tools.py.data/distro_tools_pci.yaml
+avocado-misc-tests/io/driver/driver_bind_test.py avocado-misc-tests/io/driver/driver_bind_test.py.data/driver_bind_test.yaml
+avocado-misc-tests/io/driver/module_unload_load.py avocado-misc-tests/io/driver/module_unload_load.py.data/module_unload_load.yaml
```
Comment (abdhaleegit) [inline config/tests/host/io_network.cfg:5]: please move driver module and bindunbind at last of the bucket

**Review by abdhaleegit — CHANGES_REQUESTED** [review]

**Review by spoorthys1303 — APPROVED** [review]
LGTM

**Pull request merged**